text/0032-dust-protection/0032-dust-protection.md

```diff
@@ -0,0 +1,114 @@
+                                        <td><b>Name</b></td>
+                                        <td><b>Type</b></td>
+                                        <td><b>Description</b></td>
+                                    </tr>
+                                    <tr>
+                                        <td>Output Type</td>
+                                        <td>uint8</td>
+                                        <td>
+                                            Set to <strong>value 1</strong> to denote a <i>SigLockedDustAllowanceOutput</i>.
+                                        </td>
+                                    </tr>
+                                    <tr>
+                                        <td valign="top">Address </td>
+                                        <td colspan="2">
+                                                                     <details>
+                                                <summary>Ed25519 Address</summary>
+                                                <table>
+                                                    <tr>
+                                                        <td><b>Name</b></td>
+                                                        <td><b>Type</b></td>
+                                                        <td><b>Description</b></td>
+                                                    </tr>
+                                                    <tr>
+                                                        <td>Address Type</td>
+                                                        <td>uint8</td>
+                                                        <td>
+                                                            Set to <strong>value 1</strong> to denote an <i>Ed25519 Address</i>.
+                                                        </td>
+                                                    </tr>
+                                                    <tr>
+                                                        <td>Address</td>
+                                                        <td>Array&lt;byte&gt;[32]</td>
+                                                        <td>The raw bytes of the Ed25519 address which is a BLAKE2b-256 hash of the Ed25519 public key.</td>
+                                                    </tr>
+                                                </table>
+                                            </details>
+                                        </td>
+                                    </tr>
+                                    <tr>
+                                        <td>Amount</td>
+                                        <td>uint64</td>
+                                        <td>The amount of tokens to deposit with this <i>SigLockedSingleOutput</i> output.</td>
+                                    </tr>
+                                </table>
+
+
+### The new protocol dust rule
```

Comment:
I think this validation rules should be rewritten more technically, even in pseudo-code; as there are some ambiguities on the order of processing, especially in the context of WhiteFlag.

Reply:
I am not even sure, whether we need explain this in the context of WhiteFlag. The transaction RFC only talks about "Semantic validation" of transactions, and semantically invalid transactions are handled via white flag. So I think it would be best to just describe the dust validation in the same way splitting up into Syntactical Validation and Semantic Validation and then refer to the transaction RFC

Reply:
Pseudo code added